### PR TITLE
Create model card for asafaya/bert-mini-arabic

### PR DESCRIPTION
**Pull request by alisafaya**

### CODE REVIEW

**Review by JetRunner — APPROVED** [review]
#5351

File: model_cards/asafaya/bert-mini-arabic/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+language: arabic
+---
```
Comment (julien-c) [inline model_cards/asafaya/bert-mini-arabic/README.md:3]:
```suggestion
datasets:
- oscar
- wikipedia
---
```

Comment (julien-c) [reply model_cards/asafaya/bert-mini-arabic/README.md:3]: (OSCAR is not currently in huggingface/nlp but should be soon!)

**Comment by julien-c** [comment]
Thanks! Added references to the datasets, feel free to update.

**Pull request merged**